content/en/blog/2019/announcing-1.1-eol/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Support for Istio 1.1 ends on September 19th, 2019
+description: Upcoming Istio 1.1 end of life announcement.
+publishdate: 2019-08-15
+attribution: The Istio Team
+release: 1.1
```

Comment:
```suggestion
release: 1.1
```

Reply:
I don't see the change?

Reply:
Delete "release: 1.1". That kind of annotation is only for release notes.